mobile/app/build.gradle

```diff
@@ -78,33 +77,35 @@ android {
         }
     }
 
-    lintOptions {
-        abortOnError false
-    }
     compileOptions {
         sourceCompatibility = '1.8'
         targetCompatibility = '1.8'
     }
+    namespace 'org.exarhteam.iitc_mobile'
+    lint {
+        abortOnError false
+    }
 
     applicationVariants.all { variant ->
         variant.resValue  "string", "application_id", variant.applicationId
         variant.getAssembleProvider().configure() {
-            it.doLast {
-                metaFDroid(variant.getPackageApplicationProvider().get().outputDirectory)
+            def outputDir = new File("${project.buildDir.absolutePath}/outputs/apk/${variant.flavorName}/${variant.buildType.name}")
```

Comment:
Won't this cause an error when building an aab instead of an apk?

Reply:
I have to test it